src/core/WakaTimeCore.ts

```diff
@@ -221,6 +228,17 @@ class WakaTimeCore {
     return items.loggingType;
   }
 
+  generateProjectFromDevSites(url: string): string | null {
+    const githubUrls = ['https://github.com', 'https://github.dev'];
```

Comment:
Let's make sure the url ends with slash:

`const githubUrls = ['https://github.com/', 'https://github.dev/'];`